include/Bezier/bezier.h

```diff
@@ -116,12 +115,12 @@ class Curve
   double length(double t1, double t2) const;
 
   /*!
-   * \brief Compute parameter t which is S distance from given t
+   * \brief Compute parameter t which is dS distance from given t
```

Comment:
Given an initial parameter t and its corresponding point p on the curve, compute t' of a point p' whose distance from p is dS (measured along the curve in the direction of increasing t).